android/tangram/jni/jniExports.cpp

```diff
@@ -118,6 +118,25 @@ extern "C" {
         jniEnv->ReleaseStringUTFChars(path, cPath);
     }
 
+    JNIEXPORT void JNICALL Java_com_mapzen_tangram_MapController_nativeLoadSceneWithUpdates(JNIEnv* jniEnv, jobject obj, jlong mapPtr, jstring path, jobjectArray jsceneUpdatePaths, jobjectArray jsceneUpdateValues) {
```

Comment:
Seems like we could merge this with the method above and just pass a null array when no updates are given.